MozziConfigValues.h

```diff
@@ -0,0 +1,44 @@
+/** This file keeps a list of named configuration values. 
```

Comment:
Have to say that, for the time being, I do not really get why this file is useful, I might be a bit slow… Is it anticipating for the single compilation unit?

Reply:
In a way, yes: With a single compilation unit, it will be possible (but not necessary) to specify config options directly at the top of a sketch. Since that needs to happen *before* including MozziGuts.h, we need some place to put the "labels" for the options.